src/bin/pg_autoctl/cli_get_set_properties.c

```diff
@@ -92,17 +102,21 @@ CommandLine get_commands =
 static CommandLine set_node_replication_quorum_command =
 	make_command("replication-quorum",
 				 "set replication-quorum property on the monitor",
-				 CLI_PGDATA_USAGE "<true|false>",
-				 CLI_PGDATA_OPTION,
-				 cli_getopt_pgdata,
+				 " [ --pgdata ] [ --json ] [ --name ] <true|false>",
+				 "  --pgdata      path to data directory\n"
+				 "  --name        pg_auto_failover node name\n"
+				 "  --json        output data in the JSON format\n",
+				 cli_get_set_properties_getopts,
 				 cli_set_node_replication_quorum);
 
 static CommandLine set_node_candidate_priority_command =
 	make_command("candidate-priority",
 				 "set candidate property on the monitor",
-				 CLI_PGDATA_USAGE "<priority: 0..100>",
-				 CLI_PGDATA_OPTION,
-				 cli_getopt_pgdata,
+				 " [ --pgdata ] [ --json ] [ --name ] <priority: 0..100>",
```

Comment:
Seems like this one still needs `--formation`

Reply:
Fixed.